src/components/navbar/actions.vue

```diff
@@ -38,14 +38,12 @@ except according to the terms contained in the LICENSE file.
 </template>
 
 <script>
-import request from '../../mixins/request';
 import { logOut } from '../../util/session';
 import { noop } from '../../util/util';
 import { useRequestData } from '../../request-data';
 
 export default {
   name: 'NavbarActions',
-  mixins: [request()],
```

Comment:
This component didn't actually use the mixin, so I just removed it. I think what happened here is that the component used to use the mixin when it was the one to send the logout request. At some point, that request was moved to the `logOut()` function, which this component calls, but the mixin wasn't removed at the same time.